tests/drivers/gpio/gpio_basic_api/st_arduino.overlay

```diff
@@ -4,13 +4,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-/* loopback with arduino header D0 and D1 */
+/* loopback with arduino header D9 and D10 */
 
 / {
 	resources {
 		compatible = "test-gpio-basic-api";
 		status = "okay";
-		out-gpios = <&arduino_header 6 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL | GPIO_PULL_UP)>;
-		in-gpios = <&arduino_header 7 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL | GPIO_PULL_UP)>;
+		out-gpios = <&arduino_header 15 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL | GPIO_PULL_UP)>;
+		in-gpios = <&arduino_header 16 (GPIO_ACTIVE_HIGH | GPIO_PUSH_PULL | GPIO_PULL_UP)>;
```

Comment:
No need for a specific commit. This should be fixed directly in the commit that introduced the file.

Reply:
ok